src/components/Todo/Todo.js

```diff
@@ -1,28 +1,42 @@
 import '../style/Style.css';
-import { useSelector } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
+import { addBook, removeBook } from '../../redux/books/bookSlice';
 
 const MyTodo = () => {
   const { bookData } = useSelector((store) => store.user);
+  const dispatch = useDispatch();
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const newBook = {
+      id: formData.get('item_id'),
```

Comment:
- [ ] Good job connecting the redux store with React components. :+1:  I believe there are some issues here and there. Let's work on them. On this line, you are trying to assign a value from the form with the name `item_id` as the `id` attribute of the new book. But `item_id` field does not exist on your form which results in assigning `null` value as the `id` of the book. I suggest you use some unique random value as the id to make it easier to delete the book later. 🤝️
- [ ] The `newBook` you are passing here has `id` as an attribute while the code in your redux store tries to utilize `item_id` to delete a book. I highly suggest you be consistent with your naming. 🥸️
![image](https://user-images.githubusercontent.com/17243325/231551472-779efeb2-8fbf-40a1-a87c-593d9a83e003.png)